documentation/book/proc-kafka-connect-mounting-volumes.adoc

```diff
@@ -0,0 +1,89 @@
+// This assembly is included in the following assemblies:
+//
+// assembly-kafka-connect-external-configuration.adoc
+
+[id='proc-kafka-connect-mounting-volumes-{context}']
+
+= Mounting Secrets as volumes
+
+This procedure shows how to use an {ProductPlatformName} Secret, mount it as volume into Kafka Connect, and use it to configure Kafka Connect connector.
```

Comment:
_Please reword the intro:_

You can create an {ProductPlatformName} Secret, mount it as a volume to Kafka Connect, and then use it to configure a Kafka Connect connector.